constellations/gemini-user/src/internship/internship.sass

```diff
@@ -42,4 +44,40 @@
             text-align: left       
 
     #last-paragraph
-        margin-bottom: 300px 
\ No newline at end of file
+        margin-bottom: 300px
+
+    @media only screen and (max-width: 700px)
+        & h1, p
+            text-align: center
+
+        .container-main
+            display: flex
+            flex-direction: column
+            align-items: center
+
+            & img
+                margin-left: auto
+                margin-right: auto
+                margin-bottom: 0
+                width: 90%
+        
+            & p
+                margin: 5%
+
+        .intern-main
+            display: flex
+            flex-direction: column
+            align-items: center
+            border: 0.5px solid $colorGrayLight
+
+            & h4
+                margin: 2%
+                color: $colorTurquoiseLight
+                text-align: center
+
+            & p
+                margin: 5%
```

Comment:
I think to make the small view a lot simpler, just use `margin-bottom: ____px` to space grouped elements apart. So maybe a margin-bottom for each of the `<p>` elements. And if you need more spacing after the h4, use margins.

Reply:
You shouldn't have to use `display: flex` here.